vehicles.lua

```diff
@@ -1059,6 +1059,9 @@ function courseplay:getVehicleTurnRadius(vehicle)
 		if vehicle.maxTurningRadius then
 			turnRadius = vehicle.maxTurningRadius;
 			courseplay:debug(('%s -> TurnRadius: Using Giants maxTurningRadius: %.2fm'):format(nameNum(vehicle), vehicle.maxTurningRadius), courseplay.DBG_IMPLEMENTS);
+		elseif g_vehicleConfigurations:get(vehicle, 'turnRadius') then
```

Comment:
Also, this function is obsolete and will be removed, we want the radius calculation on one place, AIDriverUtil.getTurningRadius().